### ISSUE TITLE
Manual backport of cvodes projection feature (6.2.0) to 5.7.0

### ISSUE THREAD

**Issue by mottelet** [LLNL/sundials#142]
Hello,

Would it be easy to manually backport cvodes projection feature (6.2.0) to 5.7.0, as I do no want to upgrade all my sources to the new context style ? 

S.

**Comment by balos1** [comment]
Yes, it should be possible to do that. @gardner48 can chime in on how "easy" it might be. For what its worth, [this script](https://github.com/LLNL/sundials/blob/main/scripts/upgrade-to-sundials-6-from-5.sh) can help upgrade your sources to SUNDIALS 6. If your code is not multi-threaded you can run the script, create a global `SUNContext` and then just find/replace `SUNCTX_PLACEHOLDER` with whatever you name the `SUNContext` variable.

**Comment by mottelet** [comment]
OK. Do you mean that just creating a global SUNContext and passing the variable directly (without allocating the context with SUNContext_Create() before) should be enough (if the code is not multithreaded, of course) ?

S.

**Comment by mottelet** [comment]
> > OK. Do you mean that just creating a global SUNContext and passing the variable directly (without allocating the context with SUNContext_Create() before) should be enough (if the code is not multithreaded, of course) ?
> > S.
> 
> Yes, that is correct.

OK thanks.
